webxrlayers-1.bs

```diff
@@ -1386,6 +1386,54 @@ when the [=XR Compositor=] can no longer reproject the layer.
 
 The author SHOULD redraw the content of the layer at the next [=XR animation frame=]. The event must be of type {{XRLayerEvent}}.
 
+Hit testing {#hittesting}
+===========
+
+XRLayerHitResult {#xrlayerhitresulttype}
+----------------
+
+<pre class="idl">
+interface XRLayerHitResult {
+  XRCompositionLayer layer;
+  unsigned long x;
+  unsigned long y;
+  DOMPointInit position;
+};
+</pre>
+
+The <dfn dict-member for="XRLayerHitResult">layer</dfn> member defines what layer was hit.
+
+The <dfn dict-member for="XRLayerHitResult">x</dfn> and <dfn dict-member for="XRLayerHitResult">y</dfn> member define the x and y position
+where the layer was hit.
+
+The <dfn dict-member for="XRLayerHitResult">position</dfn> member defines the position in 3D space where the hit occured.
```

Comment:
relative to which base space?

Perhaps we need a third base space argument to the query. This does seem to be getting unwieldy, thoughts @toji ?

Reply:
Maybe this can be skipped? It does seem useful to know though...

Reply:
How can you skip it? All coordinate values are given relative to some space; unlike many native APIs webxr has no underlying "native coordinate space", you can only talk about things in relation to other spaces (usually reference spaces)

There needs to be some choice of space; it could be the space used in the query though that's only really useful for truncating a ray.

Reply:
I updated the PR so a space is returned.